bedrock/products/templates/products/landing.html

```diff
@@ -15,136 +16,214 @@
+      <p><a class="mzp-c-button mzp-t-product" href="https://monitor.mozilla.org/{{ referrals }}" data-cta-type="link" data-cta-text="Check for breaches">{% if LANG == 'en-US' %}Check for breaches now{% else %}{{ ftl('firefox-products-check-for-breaches') }}{% endif %} {{ icon_external|safe }}</a></p>
+    {% endcall %}
+
+    <!-- VPN -->
+    {% call picto(
+      body=True,
+      image=resp_img(
+        url='protocol/img/logos/mozilla/vpn/logo.svg',
+        optional_attributes={
+          'class': 'mzp-c-picto-image',
+          'width': '64',
+          'height': '64',
+          'loading': 'lazy'
+        }
+      ),
+    ) %}
+      <h3 class="mzp-c-picto-heading"><a href="{{ url('products.vpn.landing') }}" data-cta-type="link" data-cta-text="VPN" data-cta-position="heading">{{ ftl('firefox-products-mozilla-vpn') }}</a></h3>
       <p>{{ ftl('firefox-products-surf-stream-and-get-work-done') }}</p>
-      <p><a class="mzp-c-cta-link" href="{{ url('products.vpn.landing') }}" rel="external noopener" data-cta-type="link" data-cta-text="Get Mozilla VPN">{{ ftl('firefox-products-get-mozilla-vpn') }}</a></p>
-    </div>
-  {% endif %}
-
-    <div class="c-landing-grid-item">
-      <img src="{{ static('img/products/landing/pocket.svg') }}" class="c-landing-grid-img" alt="" width="350" height="247">
-      <h2 class="c-landing-grid-title"><a href="https://getpocket.com/firefox_learnmore/{{ referrals }}">{{ ftl('firefox-products-pocket') }}</a></h2>
+      <p><a class="mzp-c-button mzp-t-product" href="{{ url('products.vpn.landing') + '#pricing' }}" rel="external noopener" data-cta-type="link" data-cta-text="Get VPN">{{ ftl('firefox-products-get-mozilla-vpn') }}</a></p>
+    {% endcall %}
+
+    <!-- MDN Plus -->
+    {% call picto(
+      body=True,
+      image=resp_img(
+        url='img/logos/mdn/mdn-plus-logo.svg',
+        optional_attributes={
+          'class': 'mzp-c-picto-image',
+          'width': '64',
+          'height': '64',
+          'loading': 'lazy'
+        }
+      ),
+    ) %}
+    <h3 class="mzp-c-picto-heading"><a href="https://developer.mozilla.org/{{ referrals }}" data-cta-type="link" data-cta-text="MDN Plus" data-cta-position="heading">{{ ftl('firefox-products-mdn-plus') }}</a></h3>
+      <p>{{ ftl('firefox-products-resources-for-developers') }}</p>
+      <p>
+        <a href="https://developer.mozilla.org/plus/{{ referrals }}" rel="external noopener" class="mzp-c-button mzp-t-product" data-cta-type="link" data-cta-text="Support MDN">{{ ftl('firefox-products-support-mdn') }} {{ icon_external|safe }}</a>
+      </p>
+    {% endcall %}
+
+    <!-- Thunderbird -->
+    {% call picto(
+      body=True,
+      image=resp_img(
+        url='img/logos/thunderbird/logo.svg',
```

Comment:
FYI there's now `logo-thunderbird.svg` in the same location, since yesterday;) for this exact use if needed.